proto/types/blockchain.proto

```diff
@@ -17,17 +17,14 @@ import "google/protobuf/timestamp.proto";
 // header of a block
 message BlockHeader {
   uint32 version = 1;
-  uint32 chainID = 2;
```

Comment:
Note that this introduce a breaking change to the protocol, meaning this version doesn't work with previous ones. At least we should label this as a protocol-breaking change.